src/modules/esl-panel/README.md

```diff
@@ -6,16 +6,33 @@ Version: *1.0.0-beta*.
 Authors: *Julia Murashko*.
 
 ESLPanel - is a custom element, that is used as a wrapper for content that 
-can be shown/hidden or collapsed (animated height change).
+can be shown/hidden or collapsed (height animation).
```

Comment:
It seems to me that this option is slightly annoying. Try this:

ESLPanel - is a custom element, which is used as a wrapper for content that can be shown/hidden, or collapsed (height animation).

Reply:
updated